$NetBSD: patch-CMakeLists.txt,v 1.4 2025/12/27 22:17:02 adam Exp $

Don't build ninja binary. Just install a link to the pkgsrc copy in the package
Makefile instead.

--- CMakeLists.txt.orig	2022-11-09 12:37:21.000000000 +0000
+++ CMakeLists.txt
@@ -23,7 +23,6 @@ message(STATUS "")
 message(STATUS "  RUN_NINJA_TEST : ${RUN_NINJA_TEST}")
 message(STATUS "************************************")
 
-add_subdirectory(ninja-upstream)
 
 if(RUN_NINJA_TEST)
   add_custom_target(
@@ -33,5 +32,5 @@ if(RUN_NINJA_TEST)
   )
 endif()
 
-install(TARGETS ninja COMPONENT python DESTINATION "${SKBUILD_SCRIPTS_DIR}")
+set(ninja_SOURCE_DIR "${CMAKE_SOURCE_DIR}/ninja-upstream")
 install(FILES "${ninja_SOURCE_DIR}/misc/ninja_syntax.py" COMPONENT python DESTINATION ninja)
